# Danish translation for update-manager
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the update-manager package.
# Anders Jenbo <anders@jenbo.dk>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2011-10-18 03:11+0000\n"
"PO-Revision-Date: 2011-09-09 11:45+0000\n"
"Last-Translator: Anders Feder <Unknown>\n"
"Language-Team: Danish <dansk@dansk-gruppen.dk>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2011-10-18 14:53+0000\n"
"X-Generator: Launchpad (build 14157)\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:416 ../UpdateManager/Core/utils.py:416
msgid "0 kB"
msgstr "0 kB"

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:419 ../UpdateManager/Core/utils.py:419
msgid "1 kB"
msgstr "1 kB"

#. TRANSLATORS: download size of small updates, e.g. "250 kB"
#: ../DistUpgrade/utils.py:422 ../UpdateManager/Core/utils.py:422
#, python-format
msgid "%.0f kB"
msgstr "%.0f kB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:425 ../UpdateManager/Core/utils.py:425
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:210 ../DistUpgrade/distro.py:438
#, python-format
msgid "Server for %s"
msgstr "Server for %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:228 ../DistUpgrade/distro.py:234
#: ../DistUpgrade/distro.py:250
msgid "Main server"
msgstr "Hovedserver"

#: ../DistUpgrade/distro.py:254
msgid "Custom servers"
msgstr "Brugerdefinerede servere"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:141
msgid "Could not calculate sources.list entry"
msgstr "Kunne ikke beregne sources.list-linje"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:244
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Kunne ikke finde nogen pakkefiler, måske er dette ikke en Ubuntu-disk eller "
"også er det en forkert arkitektur?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:287
msgid "Failed to add the CD"
msgstr "Kunne ikke tilføje cd"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:288
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Der opstod en fejl ved tilføjelse af cd'en, opgraderingen afbrydes. "
"Rapportér venligst dette som en fejl, hvis dette er en gyldig Ubuntu-cd.\n"
"\n"
"Fejlmeddelelsen var:\n"
"\"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:138
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Fjern pakke som er i en dårlig tilstand"
msgstr[1] "Fjern pakker som er i en dårlig tilstand"

#: ../DistUpgrade/DistUpgradeCache.py:141
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Pakken \"%s\" er usammenhængende og skal geninstalleres, men intet arkiv "
"kunne findes til denne. Ønsker du at fjerne denne pakke nu for at fortsætte?"
msgstr[1] ""
"Pakkerne \"%s\" er usammenhængende og skal geninstalleres, men intet arkiv "
"kunne findes til disse. Ønsker du at fjerne disse pakker nu for at fortsætte?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:239
msgid "The server may be overloaded"
msgstr "Serveren er muligvis overbelastet"

#: ../DistUpgrade/DistUpgradeCache.py:352
msgid "Broken packages"
msgstr "Ødelagte pakker"

#: ../DistUpgrade/DistUpgradeCache.py:353
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Dit system indeholder ødelagte pakker, som ikke kan repareres med dette "
"program. Reparér dem venligst med synaptic eller apt-get før du fortsætter."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:648
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Et uløseligt problem opstod under beregning af opgraderingen:\n"
"%s\n"
"\n"
"Det kan ske, hvis man:\n"
" * Opgraderer til en udviklingsudgave af Ubuntu\n"
" * Kører den nuværende udviklingsudgave af Ubuntu\n"
" * Bruger uofficielle software-pakker, der ikke leveres af Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:658
msgid "This is most likely a transient problem, please try again later."
msgstr "Dette er højst sandsynligt et forbigående problem, prøv igen senere."

#: ../DistUpgrade/DistUpgradeCache.py:661
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Hvis intet af dette passer, så indrapporter venligst denne fejl.  Dette kan "
"gøres ved at køre kommandoen \"ubuntu-bug update-manager\" i en terminal."

#: ../DistUpgrade/DistUpgradeCache.py:666
#: ../UpdateManager/UpdateManager.py:1023
msgid "Could not calculate the upgrade"
msgstr "Kunne ikke beregne opgraderingen"

#: ../DistUpgrade/DistUpgradeCache.py:717
msgid "Error authenticating some packages"
msgstr "Fejl ved godkendelse af nogle pakker"

#: ../DistUpgrade/DistUpgradeCache.py:718
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Nogle pakker kunne ikke godkendes. Dette kan skyldes et forbigående problem "
"med netværket. Det anbefales, at du prøver igen senere. Se længere nede "
"listen over pakker som ikke kunne godkendes."

#: ../DistUpgrade/DistUpgradeCache.py:738
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Pakken \"%s\" er markeret til fjernelse, men den er i listen over pakker, "
"der ikke må fjernes."

#: ../DistUpgrade/DistUpgradeCache.py:742
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Den nødvendige pakke \"%s\" er markeret til fjernelse."

#: ../DistUpgrade/DistUpgradeCache.py:751
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Forsøger at installere sortlistet version \"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:869
#, python-format
msgid "Can't install '%s'"
msgstr "Kan ikke installere \"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:870
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Det var umuligt at installere en påkrævet pakke. Rapportér venligst denne "
"fejl ved hjælp af \"ubuntu-bug update-manager\" i en terminal."

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:881
msgid "Can't guess meta-package"
msgstr "Kan ikke gætte meta-pakke"

#: ../DistUpgrade/DistUpgradeCache.py:882
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Dit system indeholder ikke en ubuntu-desktop-, kubuntu-desktop-, xubuntu-"
"desktop- eller edubuntu-desktop-pakke, og det var ikke muligt at genkende, "
"hvilken Ubuntu-udgave du kører.\n"
"\n"
"Installér venligst en af ovenfor nævnte pakker ved hjælp af synaptic eller "
"apt-get, før du fortsætter."

#: ../DistUpgrade/DistUpgradeController.py:100
msgid "Reading cache"
msgstr "Læser cache"

#: ../DistUpgrade/DistUpgradeController.py:205
msgid "Unable to get exclusive lock"
msgstr "Kan ikke opnå eksklusiv lås"

#: ../DistUpgrade/DistUpgradeController.py:206
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Det skyldes sandsynligvis, at et andet pakkehåndterings-program (såsom apt-"
"get eller aptitude) allerede kører. Luk venligst det program først."

#: ../DistUpgrade/DistUpgradeController.py:239
msgid "Upgrading over remote connection not supported"
msgstr "Opgradering over fjernforbindelse er ikke understøttet"

#: ../DistUpgrade/DistUpgradeController.py:240
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Du kører opgraderingen over en ekstern ssh-forbindelse med en grænseflade, "
"der ikke understøtter dette. Prøv en teksttilstandopgradering med \"do-"
"release-upgrade\".\n"
"\n"
"Opgraderingen vil afbryde nu. Prøv uden ssh."

#: ../DistUpgrade/DistUpgradeController.py:254
msgid "Continue running under SSH?"
msgstr "Forsæt med at køre over SSH?"

#: ../DistUpgrade/DistUpgradeController.py:255
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Denne session kører over ssh. Det er ikke anbefalet at udføre en opgradering "
"over ssh i øjeblikket, da det i tilfælde af en fejl er sværere at genskabe.\n"
"\n"
"Hvis du vil fortsætte, vil en ekstra ssh-tjeneste blive startet på port "
"\"%s\"\n"
"Vil du fortsætte?"

#: ../DistUpgrade/DistUpgradeController.py:269
msgid "Starting additional sshd"
msgstr "Starter ekstra sshd"

#: ../DistUpgrade/DistUpgradeController.py:270
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"For at sikre nemmere genetablering i tilfælde af fejl, vil en ekstra sshd "
"blive startet på port \"%s\". Hvis noget går galt med den kørende ssh, kan "
"du stadig forbinde til den ekstra.\n"

#: ../DistUpgrade/DistUpgradeController.py:278
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Hvis du har en firewall, kan det være nødvendigt midlertidigt at åbne denne "
"port. Da der potentielt kan være fare forbundet med dette, vil det ikke "
"blive gjort automatisk. Du kan åbne porten med f.eks.:\n"
"\"%s\""

#: ../DistUpgrade/DistUpgradeController.py:349
#: ../DistUpgrade/DistUpgradeController.py:394
msgid "Can not upgrade"
msgstr "Kan ikke opgradere"

#: ../DistUpgrade/DistUpgradeController.py:350
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr ""
"En opgradering fra \"%s\" til \"%s\" er ikke understøttet med dette værktøj."

#: ../DistUpgrade/DistUpgradeController.py:359
msgid "Sandbox setup failed"
msgstr "Testopsætning mislykkedes"

#: ../DistUpgrade/DistUpgradeController.py:360
msgid "It was not possible to create the sandbox environment."
msgstr "Det var ikke muligt at oprette testmiljøet."

#: ../DistUpgrade/DistUpgradeController.py:366
msgid "Sandbox mode"
msgstr "Testtilstand"

#: ../DistUpgrade/DistUpgradeController.py:367
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a systemdir from now until the next reboot are "
"permanent."
msgstr ""
"Denne opgradering kører i testtilstand. Alle ændringer skrives til \"%s\" og "
"vil gå tabt ved næste genstart.\n"
"\n"
"*Ingen* ændringer, skrevet til en systemkatalog fra nu af og indtil den "
"næste genstart, vil være permanente."

#: ../DistUpgrade/DistUpgradeController.py:395
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Din python-installation er beskadiget. Ret venligst det symbolske link i "
"\"/usr/bin/python\"."

#: ../DistUpgrade/DistUpgradeController.py:421
msgid "Package 'debsig-verify' is installed"
msgstr "Pakken \"debsig-verify\" er installeret"

#: ../DistUpgrade/DistUpgradeController.py:422
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Opgraderingen kan ikke fortsætte med den pakke installeret.\n"
"Fjern den først med synaptic eller \"apt-get remove debsig-verify\" og kør "
"så opgraderingen igen."

#: ../DistUpgrade/DistUpgradeController.py:432
msgid "Include latest updates from the Internet?"
msgstr "Inkludér de seneste opdateringer fra internettet?"

#: ../DistUpgrade/DistUpgradeController.py:433
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Opgraderingssystemet kan bruge internettet til automatisk at hente de "
"seneste opdateringer, samt installere dem. Dette anbefales kraftigt, hvis du "
"har en netværksforbindelse.\n"
"\n"
"Opgraderingen vil tage længere tid, men når den er færdig, vil dit system "
"være fuldt opdateret. Du kan vælge ikke at gøre dette, men du bør installere "
"de nyeste opdateringer snarest efter opgradering.\n"
"Hvis du svarer \"nej\" her, benyttes netværket slet ikke."

#: ../DistUpgrade/DistUpgradeController.py:653
#, python-format
msgid "disabled on upgrade to %s"
msgstr "slået fra under opgradering til %s"

#: ../DistUpgrade/DistUpgradeController.py:680
msgid "No valid mirror found"
msgstr "Ingen gyldige kilder fundet"

#: ../DistUpgrade/DistUpgradeController.py:681
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Under scanning af dine softwarekilder blev et filspejl til opgraderingen "
"ikke fundet. Dette kan ske, hvis du anvender et internt filspejl, eller hvis "
"filspejlets oplysninger er forældede.\n"
"\n"
"Vil du ændre filen \"sources.list\" alligevel? Hvis du vælger \"Ja\" her vil "
"den opdatere alle \"%s\" til \"%s\" poster.\n"
"Hvis du vælger \"Nej\" annulleres opgraderingen."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:701
msgid "Generate default sources?"
msgstr "Generér standardkilder?"

#: ../DistUpgrade/DistUpgradeController.py:702
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Under scanning af din \"sources.list\" blev ingen gyldig værdi for \"%s\" "
"fundet.\n"
"\n"
"Skal standardværdier for \"%s\" tilføjes? Hvis du vælger \"Nej\", annulleres "
"opgraderingen."

#: ../DistUpgrade/DistUpgradeController.py:737
msgid "Repository information invalid"
msgstr "Arkivinformation ugyldig"

#: ../DistUpgrade/DistUpgradeController.py:738
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug using the command 'ubuntu-bug update-manager' in a "
"terminal."
msgstr ""
"Opgradering af arkivinformation resulteret i en ugyldig fil. Rapporter "
"venligst dette som en fejl med kommandoen \"ubuntu-bug update-manager\" i en "
"terminal."

#: ../DistUpgrade/DistUpgradeController.py:746
msgid "Third party sources disabled"
msgstr "Trediepartskilder er fravalgt"

#: ../DistUpgrade/DistUpgradeController.py:747
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Nogle tredjepartselementer i din sources.list blev deaktiveret. Du kan "
"genaktivere dem efter opgraderingen med værktøjet \"Softwarekilder\" eller "
"med pakkehåndtering."

#: ../DistUpgrade/DistUpgradeController.py:787
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Pakke i inkonsistent tilstand"
msgstr[1] "Pakker i inkonsistent tilstand"

#: ../DistUpgrade/DistUpgradeController.py:790
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Pakkerne \"%s\" er usammenhængende og skal geninstalleres, men der blev ikke "
"fundet nogen tilhørende arkiver. Geninstallér venligst pakkerne manuelt "
"eller fjern dem fra systemet."
msgstr[1] ""
"Pakkerne \"%s\" er usammenhængende og skal geninstalleres, men der blev ikke "
"fundet et tilhørende arkiv. Geninstallér venligst pakkerne manuelt eller "
"fjern dem fra systemet."

#: ../DistUpgrade/DistUpgradeController.py:838
msgid "Error during update"
msgstr "Fejl under opdatering"

#: ../DistUpgrade/DistUpgradeController.py:839
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"En fejl forekom under opdateringen. Dette skyldes som regel et "
"netværksproblem, kontrollér venligst din netværksforbindelse og prøv igen."

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:848
#: ../UpdateManager/UpdateManager.py:747
msgid "Not enough free disk space"
msgstr "Der er ikke nok fri diskplads"

#: ../DistUpgrade/DistUpgradeController.py:849
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"Opgraderingen er blevet afbrudt. Opgraderingen kræver samlet %s fri "
"diskplads på disken \"%s\". Frigør venligst mindst yderligere %s diskplads "
"på \"%s\". Tøm papirkurven og fjern midlertidige pakker fra tidligere "
"installationer ved hjælp af kommandoen \"sudo apt-get clean\"."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:878
#: ../DistUpgrade/DistUpgradeController.py:1630
msgid "Calculating the changes"
msgstr "Udregner ændringerne"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:910
msgid "Do you want to start the upgrade?"
msgstr "Vil du starte opgraderingen?"

#: ../DistUpgrade/DistUpgradeController.py:976
msgid "Upgrade canceled"
msgstr "Opgradering annulleret"

#: ../DistUpgrade/DistUpgradeController.py:977
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Denne opgradering vil nu blive annulleret, og den originale tilstand for "
"systemet vil blive gendannet. Du kan genoptage opgraderingen på et senere "
"tidspunkt."

#: ../DistUpgrade/DistUpgradeController.py:983
#: ../DistUpgrade/DistUpgradeController.py:1109
msgid "Could not download the upgrades"
msgstr "Kunne ikke hente opgraderingerne"

#: ../DistUpgrade/DistUpgradeController.py:984
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Opgraderingen blev afbrudt. Kontroller venligst din internetforbindelse "
"eller installationsmedie og prøv igen. Alle filer der blev hentet indtil "
"videre er blevet gemt."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1060
#: ../DistUpgrade/DistUpgradeController.py:1097
#: ../DistUpgrade/DistUpgradeController.py:1202
msgid "Error during commit"
msgstr "Fejl under gennemførelse"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1062
#: ../DistUpgrade/DistUpgradeController.py:1099
#: ../DistUpgrade/DistUpgradeController.py:1241
msgid "Restoring original system state"
msgstr "Genskaber oprindelig systemtilstand"

#: ../DistUpgrade/DistUpgradeController.py:1063
#: ../DistUpgrade/DistUpgradeController.py:1078
#: ../DistUpgrade/DistUpgradeController.py:1100
msgid "Could not install the upgrades"
msgstr "Kunne ikke installere opgraderingerne"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1068
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Opgraderingen er blevet afbrudt. Dit system kan være ude af stand til at "
"starte. Der vil nu blive kørt en genoprettelse (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1073
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug using the command 'ubuntu-bug update-manager' in a "
"terminal and include the files in /var/log/dist-upgrade/ in the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Rapportér venligst denne fejl ved hjælp af kommandoen \"ubuntu-bug update-"
"manager\" i en terminal og inkludér filerne i /var/log/dist-upgrade/ i en "
"fejlrapport.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1110
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Opgraderingen er blevet afbrudt. Kontroller venligst din internetforbindelse "
"og installationsmedie og prøv igen. "

#: ../DistUpgrade/DistUpgradeController.py:1190
msgid "Remove obsolete packages?"
msgstr "Fjern forældede pakker?"

#: ../DistUpgrade/DistUpgradeController.py:1191
#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "_Keep"
msgstr "_Behold"

#: ../DistUpgrade/DistUpgradeController.py:1191
msgid "_Remove"
msgstr "_Fjern"

#: ../DistUpgrade/DistUpgradeController.py:1203
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Et problem opstod under oprydningen. Se venligst beskeden nedenfor for mere "
"information. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1279
msgid "Required depends is not installed"
msgstr "Påkrævede afhængigheder er ikke installeret"

#: ../DistUpgrade/DistUpgradeController.py:1280
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Den påkrævede afhængighed \"%s\" er ikke installeret. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1527
#: ../DistUpgrade/DistUpgradeController.py:1597
msgid "Checking package manager"
msgstr "Kontrollerer pakkehåndtering"

#: ../DistUpgrade/DistUpgradeController.py:1532
msgid "Preparing the upgrade failed"
msgstr "Klargøring af opgraderingen fejlede"

#: ../DistUpgrade/DistUpgradeController.py:1533
msgid ""
"Preparing the system for the upgrade failed. Please report this using the "
"command 'ubuntu-bug update-manager' in a terminal and include the files in "
"/var/log/dist-upgrade/ in the bug report."
msgstr ""
"Forberedning af systemet til en opgradering mislykkedes. Rapportér venligst "
"dette ved hjælp af kommandoen \"ubuntu-bug update-manager\" i en terminal og "
"inkludér filerne i /var/log/dist-upgrade/ i fejlrapporten."

#: ../DistUpgrade/DistUpgradeController.py:1549
msgid "Getting upgrade prerequisites failed"
msgstr "Hentning af opgraderingsforudsætninger fejlede"

#: ../DistUpgrade/DistUpgradeController.py:1550
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug using the command 'ubuntu-bug update-manager' in "
"a terminal and include the files in /var/log/dist-upgrade/ in the bug report."
msgstr ""
"Systemet var ikke i stand til at hente forudsætningerne for opgraderingen. "
"Opgraderingen vil afbryde nu og gendanne den oprindelige systemtilstand.\n"
"\n"
"Rapportér venligst dette som en fejl ved hjælp af kommandoen \"ubuntu-bug "
"update-manager\" i en terminal og inkludér filerne i /var/log/dist-upgrade/ "
"i fejlrapporten."

#: ../DistUpgrade/DistUpgradeController.py:1581
msgid "Updating repository information"
msgstr "Opdaterer arkivinformation"

#: ../DistUpgrade/DistUpgradeController.py:1588
msgid "Failed to add the cdrom"
msgstr "Fejl under tilføjelse af cd-rom"

#: ../DistUpgrade/DistUpgradeController.py:1589
msgid "Sorry, adding the cdrom was not successful."
msgstr "Beklager, tilføjelse af cd-rom'en mislykkedes."

#: ../DistUpgrade/DistUpgradeController.py:1617
msgid "Invalid package information"
msgstr "Ugyldig pakkeinformation"

#: ../DistUpgrade/DistUpgradeController.py:1618
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal and include the files in "
"/var/log/dist-upgrade/ in the bug report."
msgstr ""
"Efter din pakkeinformation blev opdateret kan den essentielle pakke \"%s\" "
"ikke længere findes.\n"
"Dette indikerer en alvorlig fejl. Rapporter venligst denne fejl med "
"kommandoen \"ubuntu-bug update manager\" i en terminal og inkluder filerne i "
"/var/log/dist-upgrade/ i fejlrapporten."

#: ../DistUpgrade/DistUpgradeController.py:1636
#: ../DistUpgrade/DistUpgradeController.py:1689
msgid "Fetching"
msgstr "Henter"

#: ../DistUpgrade/DistUpgradeController.py:1642
#: ../DistUpgrade/DistUpgradeController.py:1693
msgid "Upgrading"
msgstr "Opgraderer"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1647
#: ../DistUpgrade/DistUpgradeController.py:1695
#: ../DistUpgrade/DistUpgradeController.py:1702
#: ../DistUpgrade/DistUpgradeController.py:1713
msgid "Upgrade complete"
msgstr "Opgradering gennemført"

#: ../DistUpgrade/DistUpgradeController.py:1648
#: ../DistUpgrade/DistUpgradeController.py:1696
#: ../DistUpgrade/DistUpgradeController.py:1703
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""
"Opgraderingen er afsluttet, men der opstod fejl under opgraderingsprocessen."

#: ../DistUpgrade/DistUpgradeController.py:1655
msgid "Searching for obsolete software"
msgstr "Søger efter forældet software"

#: ../DistUpgrade/DistUpgradeController.py:1664
msgid "System upgrade is complete."
msgstr "Systemopgradering er fuldført."

#: ../DistUpgrade/DistUpgradeController.py:1714
msgid "The partial upgrade was completed."
msgstr "Den delvise opgradering er færdig."

#: ../DistUpgrade/DistUpgradeQuirks.py:201
msgid "evms in use"
msgstr "evms er i brug"

#: ../DistUpgrade/DistUpgradeQuirks.py:202
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Dit system bruger diskområdehåndteringen \"evms\" i /proc/mounts. \"evms\"-"
"softwaren understøttes ikke længere. Slå den venligst fra og kør "
"opgraderingen igen, når dette er gjort."

#: ../DistUpgrade/DistUpgradeQuirks.py:506
msgid "Your graphics hardware may not be fully supported in Ubuntu 11.04."
msgstr "Dit grafikkort er muligvis ikke fuldt understøttet i Ubuntu 11.04."

#: ../DistUpgrade/DistUpgradeQuirks.py:508
msgid ""
"The support in Ubuntu 11.04 for your intel graphics hardware is limited and "
"you may encounter problems after the upgrade. Do you want to continue with "
"the upgrade?"
msgstr ""
"Understøttelsen i Ubuntu 11.04 for din grafikhardware fra Intel er begrænset "
"og du kan støde på problemer efter opgraderingen. Ønsker du at fortsætte med "
"opgraderingen?"

#: ../DistUpgrade/DistUpgradeQuirks.py:528
#: ../DistUpgrade/DistUpgradeQuirks.py:556
#: ../DistUpgrade/DistUpgradeQuirks.py:583
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Opgradering kan reducere skrivebordseffekter, ydelse i spil og "
"grafikintensive programmer."

#: ../DistUpgrade/DistUpgradeQuirks.py:532
#: ../DistUpgrade/DistUpgradeQuirks.py:560
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Denne computer benytter i øjeblikket NVIDIA's \"nvidia\"-grafikdriver. Der "
"er ingen version af denne driver tilgængelig, som fungerer sammen med dit "
"grafikkort i Ubuntu 10.04 LTS.\n"
"\n"
"Vil du fortsætte?"

#: ../DistUpgrade/DistUpgradeQuirks.py:587
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Denne computer benytter i øjeblikket AMD's \"fglrx\"-grafikdriver. Der er "
"ingen version af denne driver tilgængelig, som fungerer sammen med dit "
"grafikkort i Ubuntu 10.04 LTS.\n"
"\n"
"Vil du fortsætte?"

#: ../DistUpgrade/DistUpgradeQuirks.py:617
msgid "No i686 CPU"
msgstr "Ikke en i686 cpu"

#: ../DistUpgrade/DistUpgradeQuirks.py:618
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Dit system bruger en i586 cpu eller en cpu, der ikke har \"cmov-"
"udvidelsen\". Alle pakker er bygget med optimeringer, der kræver i686 som et "
"minimum. Det er ikke muligt at opgradere dit system til en nyere udgave af "
"Ubuntu med denne hardware."

#: ../DistUpgrade/DistUpgradeQuirks.py:654
msgid "No ARMv6 CPU"
msgstr "Ingen ARMv6 CPU"

#: ../DistUpgrade/DistUpgradeQuirks.py:655
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Dit system bruger en ARM CPU, der er ældre end ARMv6-arkitekturen. Alle "
"pakker i karmic blev bygget med optimeringer der kræver ARMv6 som den "
"minimale arkitektur. Det er ikke muligt at opgradere dit system til en ny "
"Ubuntu-udgave med denne hardware."

#: ../DistUpgrade/DistUpgradeQuirks.py:675
msgid "No init available"
msgstr "Ingen tilgængelig init"

#: ../DistUpgrade/DistUpgradeQuirks.py:676
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Dit system synes at være et virtualiseret miljø uden en init-dæmon, f.eks. "
"Linux-VServer. Ubuntu 10.04 LTS kan ikke fungere i denne type miljø, hvilket "
"nødvendiggør en opdatering af din virtuelle maskines konfiguration.\n"
"\n"
"Er du sikker på at du vil fortsætte?"

#: ../DistUpgrade/DistUpgradeMain.py:45
msgid "Sandbox upgrade using aufs"
msgstr "Kør opgraderingen i et lukket miljø via aufs"

#: ../DistUpgrade/DistUpgradeMain.py:47
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Brug den angivne sti til at søge efter en cdrom med pakker der kan opgraderes"

#: ../DistUpgrade/DistUpgradeMain.py:53
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Brug brugerinterface. Tilgængelige i øjeblikket: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:56
msgid "*DEPRECATED* this option will be ignored"
msgstr "*UDFASET* dette valg vil blive ignoreret"

#: ../DistUpgrade/DistUpgradeMain.py:59
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "Udfør kun en delvis opgradering (ingen genskrivning af sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:62
msgid "Disable GNU screen support"
msgstr "Slå understøttelse af GNU screen fra."

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid "Set datadir"
msgstr "Indstil datamappe"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:117
#: ../DistUpgrade/DistUpgradeViewKDE.py:189
#: ../UpdateManager/DistUpgradeFetcherKDE.py:147
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Indsæt venligst \"%s\" i drevet \"%s\""

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:138
#: ../DistUpgrade/DistUpgradeViewKDE.py:203
msgid "Fetching is complete"
msgstr "Hentning er gennemført"

#: ../DistUpgrade/DistUpgradeViewGtk.py:146
#: ../DistUpgrade/DistUpgradeViewGtk3.py:149
#: ../DistUpgrade/DistUpgradeViewKDE.py:216
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Henter fil %li ud af %li ved %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk.py:295
#: ../DistUpgrade/DistUpgradeViewGtk3.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:301
#: ../DistUpgrade/DistUpgradeViewKDE.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:361
#, python-format
msgid "About %s remaining"
msgstr "Omkring %s tilbage"

#: ../DistUpgrade/DistUpgradeViewGtk.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:155
#: ../DistUpgrade/DistUpgradeViewKDE.py:219
#, python-format
msgid "Fetching file %li of %li"
msgstr "Henter fil %li af %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:183
#: ../DistUpgrade/DistUpgradeViewGtk3.py:186
#: ../DistUpgrade/DistUpgradeViewKDE.py:253
msgid "Applying changes"
msgstr "Udfører ændringer"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:208
#: ../DistUpgrade/DistUpgradeViewGtk3.py:211
#: ../DistUpgrade/DistUpgradeViewKDE.py:266
msgid "dependency problems - leaving unconfigured"
msgstr "afhængighedsproblemer - forlader ukonfigureret"

#: ../DistUpgrade/DistUpgradeViewGtk.py:213
#: ../DistUpgrade/DistUpgradeViewGtk3.py:216
#: ../DistUpgrade/DistUpgradeViewKDE.py:268
#, python-format
msgid "Could not install '%s'"
msgstr "Kunne ikke installere \"%s\""

#: ../DistUpgrade/DistUpgradeViewGtk.py:214
#: ../DistUpgrade/DistUpgradeViewGtk3.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:269
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Opgraderingen vil fortsætte, men pakken \"%s\" er muligvis ikke i en "
"fungerende tilstand. Overvej at indsende en fejlrapport om det."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:231
#: ../DistUpgrade/DistUpgradeViewGtk3.py:234
#: ../DistUpgrade/DistUpgradeViewKDE.py:290
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Udskift den tilpassede konfigurationsfil\n"
"\"%s\"?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:232
#: ../DistUpgrade/DistUpgradeViewGtk3.py:235
#: ../DistUpgrade/DistUpgradeViewKDE.py:291
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Du mister alle ændringer, du har lavet til denne konfigurationsfil, hvis du "
"vælger at erstatte den med en nyere udgave."

#: ../DistUpgrade/DistUpgradeViewGtk.py:250
#: ../DistUpgrade/DistUpgradeViewGtk3.py:253
#: ../DistUpgrade/DistUpgradeViewKDE.py:313
msgid "The 'diff' command was not found"
msgstr "Kommandoen \"diff\" blev ikke fundet"

#: ../DistUpgrade/DistUpgradeViewGtk.py:463
#: ../DistUpgrade/DistUpgradeViewGtk3.py:469
#: ../DistUpgrade/DistUpgradeViewText.py:90
msgid "A fatal error occurred"
msgstr "En alvorlig fejl opstod"

#: ../DistUpgrade/DistUpgradeViewGtk.py:464
#: ../DistUpgrade/DistUpgradeViewGtk3.py:470
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Rapporter venligst dette som en fejl (hvis du ikke allerede har gjort det) "
"og inkluder filerne /var/log/dist-upgrade/main.log og /var/log/dist-"
"upgrade/apt.log i din rapport. Opgraderingen er blevet afbrudt.\n"
"Din originale sources.list blev gemt i /etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:481
#: ../DistUpgrade/DistUpgradeViewGtk3.py:487
msgid "Ctrl-c pressed"
msgstr "Ctrl-c trykket"

#: ../DistUpgrade/DistUpgradeViewGtk.py:482
#: ../DistUpgrade/DistUpgradeViewGtk3.py:488
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Dette vil afbryde handlingen og kan efterlade systemet i en ødelagt "
"tilstand. Er du sikker på at du vil gøre det?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:630
#: ../DistUpgrade/DistUpgradeViewGtk3.py:621
msgid "To prevent data loss close all open applications and documents."
msgstr "Luk alle åbne programmer og dokumenter for at undgå tab af data."

#: ../DistUpgrade/DistUpgradeViewGtk.py:644
#: ../DistUpgrade/DistUpgradeViewGtk3.py:635
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Ikke længere understøttet af Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:645
#: ../DistUpgrade/DistUpgradeViewGtk3.py:636
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Nedgrader (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:646
#: ../DistUpgrade/DistUpgradeViewGtk3.py:637
#, python-format
msgid "Remove (%s)"
msgstr "Fjern (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:638
#, python-format
msgid "No longer needed (%s)"
msgstr "Ikke længere nødvendig (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:639
#, python-format
msgid "Install (%s)"
msgstr "Installér (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:640
#, python-format
msgid "Upgrade (%s)"
msgstr "Opgrader (%s)"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:190
#: ../UpdateManager/DistUpgradeFetcherKDE.py:149
msgid "Media Change"
msgstr "Medieskift"

#: ../DistUpgrade/DistUpgradeViewKDE.py:325
msgid "Show Difference >>>"
msgstr "Vis forskel >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:328
msgid "<<< Hide Difference"
msgstr "<<< Skjul forskel"

#: ../DistUpgrade/DistUpgradeViewKDE.py:543
msgid "Error"
msgstr "Fejl"

#: ../DistUpgrade/DistUpgradeViewKDE.py:555
msgid "&Cancel"
msgstr "&Annullér"

#: ../DistUpgrade/DistUpgradeViewKDE.py:557
#: ../DistUpgrade/DistUpgradeViewKDE.py:799
msgid "&Close"
msgstr "&Luk"

#: ../DistUpgrade/DistUpgradeViewKDE.py:603
msgid "Show Terminal >>>"
msgstr "Vis terminal >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:606
msgid "<<< Hide Terminal"
msgstr "<<< Skjul terminal"

#: ../DistUpgrade/DistUpgradeViewKDE.py:686
msgid "Information"
msgstr "Oplysninger"

#: ../DistUpgrade/DistUpgradeViewKDE.py:737
#: ../DistUpgrade/DistUpgradeViewKDE.py:782
#: ../DistUpgrade/DistUpgradeViewKDE.py:785 ../DistUpgrade/DistUpgrade.ui.h:11
msgid "Details"
msgstr "Detaljer"

#: ../DistUpgrade/DistUpgradeViewKDE.py:763
#, python-format
msgid "No longer supported %s"
msgstr "Ikke længere understøttet %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:765
#, python-format
msgid "Remove %s"
msgstr "Fjern %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:767
#: ../DistUpgrade/DistUpgradeViewText.py:179
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Fjern (var installeret automatisk) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:769
#, python-format
msgid "Install %s"
msgstr "Installér %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:771
#, python-format
msgid "Upgrade %s"
msgstr "Opgradér %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:795
#: ../DistUpgrade/DistUpgradeViewText.py:227
msgid "Restart required"
msgstr "Genstart påkrævet"

#: ../DistUpgrade/DistUpgradeViewKDE.py:795
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Genstart systemet for at fuldføre opgraderingen</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:798 ../DistUpgrade/DistUpgrade.ui.h:25
#: ../data/gtkbuilder/UpdateManager.ui.h:32
msgid "_Restart Now"
msgstr "_Genstart nu"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:816
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Annullér den igangværende opgradering?</big></b>\n"
"\n"
"Systemet kan ende i en ubrugelig tilstand, hvis du annullerer opgraderingen. "
"Du opfordres kraftigt til at genoptage opgraderingen."

#: ../DistUpgrade/DistUpgradeViewKDE.py:820
msgid "Cancel Upgrade?"
msgstr "Afbryd Opgradering?"

#: ../DistUpgrade/DistUpgradeView.py:59
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li dag"
msgstr[1] "%li dage"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li time"
msgstr[1] "%li timer"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minut"
msgstr[1] "%li minutter"

#: ../DistUpgrade/DistUpgradeView.py:64
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li sekund"
msgstr[1] "%li sekunder"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:80
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:98
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:147
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Denne overførsel vil tage omkring %s med en 1Mbit DSL-forbindelse og omkring "
"%s med et 56k modem."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:151
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Denne overførsel vil tage omkring %s med din forbindelse. "

#: ../DistUpgrade/DistUpgradeView.py:253 ../DistUpgrade/DistUpgrade.ui.h:16
msgid "Preparing to upgrade"
msgstr "Gør klar til opgradering"

#: ../DistUpgrade/DistUpgradeView.py:254
msgid "Getting new software channels"
msgstr "Indhenter nye softwarekanaler"

#: ../DistUpgrade/DistUpgradeView.py:255 ../DistUpgrade/DistUpgrade.ui.h:14
msgid "Getting new packages"
msgstr "Henter nye pakker"

#: ../DistUpgrade/DistUpgradeView.py:256 ../DistUpgrade/DistUpgrade.ui.h:15
msgid "Installing the upgrades"
msgstr "Installerer opgraderingerne"

#: ../DistUpgrade/DistUpgradeView.py:257 ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Cleaning up"
msgstr "Rydder op"

#: ../DistUpgrade/DistUpgradeView.py:342
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d installeret pakke understøttes ikke længere af Canonical. Du kan "
"stadig få support fra fællesskabet."
msgstr[1] ""
"%(amount)d installerede pakker understøttes ikke længere af Canonical. Du "
"kan stadig få support fra fællesskabet."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:351
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d pakke vil blive fjernet."
msgstr[1] "%d pakker vil blive fjernet."

#: ../DistUpgrade/DistUpgradeView.py:356
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d ny pakke vil blive installeret."
msgstr[1] "%d nye pakker vil blive installeret."

#: ../DistUpgrade/DistUpgradeView.py:362
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d pakke vil blive opgraderet."
msgstr[1] "%d pakker vil blive opgraderet."

#: ../DistUpgrade/DistUpgradeView.py:367
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Du skal i alt hente %s. "

#: ../DistUpgrade/DistUpgradeView.py:371
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be cancelled."
msgstr ""
"Hentning og installation af opgraderingen kan tage flere timer og kan ikke "
"afbrydes senere. Når hentningen er færdig, kan processen ikke afbrydes."

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:377 ../UpdateManager/UpdateManager.py:664
msgid "Your system is up-to-date"
msgstr "Dit system er opdateret"

#: ../DistUpgrade/DistUpgradeView.py:378
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Der er ingen opgraderinger tilgængelige for dit system. Opgraderingen vil nu "
"blive afbrudt."

#: ../DistUpgrade/DistUpgradeView.py:391
msgid "Reboot required"
msgstr "Genstart af maskinen er påkrævet"

#: ../DistUpgrade/DistUpgradeView.py:392
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Opgraderingen er afsluttet og et genstart af maskinen er påkrævet. Vil du "
"gøre dette nu?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:131
#: ../DistUpgrade/DistUpgradeFetcherCore.py:132
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:131
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:132
msgid "Could not run the upgrade tool"
msgstr "Kunne ikke køre opgraderingsværktøjet"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:132
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:132
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug update-manager'."
msgstr ""
"Dette er højst sandsynligt en fejl i opgraderingsværktøjet. Rapportér den "
"venligst som en fejl med kommandoen 'ubuntu-bug update-manager'."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:207
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:207
msgid "Upgrade tool signature"
msgstr "Opgradér værkstøjets signatur"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:214
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:214
msgid "Upgrade tool"
msgstr "Opgraderingsværktøj"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:248
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:248
msgid "Failed to fetch"
msgstr "Fejl ved hentning"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:249
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Hentning af opgraderingen fejlede. Dette kan skyldes et netværksproblem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:253
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:253
msgid "Failed to extract"
msgstr "Fejl ved udpakning"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:254
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:254
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Udpakning af opgraderingen fejlede. Dette kan skyldes et problem med "
"netværket eller med serveren. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:259
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:259
msgid "Verification failed"
msgstr "Verifikation mislykkedes"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:260
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:260
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Efterprøvning af opgraderingen fejlede. Det skyldes muligvis et problem med "
"netværket eller serveren. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:265
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:265
msgid "Authentication failed"
msgstr "Godkendelse mislykkedes"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:266
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:266
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Godkendelse af opgraderingen fejlede. Der er muligvis et problem med "
"netværket eller med serveren. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:280
#: ../DistUpgrade/DistUpgradeFetcherCore.py:286
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:280
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:286
msgid "Can not run the upgrade"
msgstr "Kan ikke køre opgraderingen"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:281
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:281
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Dette er normalt forårsaget af et system hvor /tmp er monteret noexec. "
"Montér venligst uden noexec og kør opgraderingen igen."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:287
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:287
#, python-format
msgid "The error message is '%s'."
msgstr "Fejlmeddelelsen er \"%s\"."

#: ../DistUpgrade/DistUpgradeViewText.py:91
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Rapporter venligst dette som en fejl og inkluder filerne /var/log/dist-"
"upgrade/main.log og /var/log/dist-upgrade/apt.log i din rapport. "
"Opgraderingen er blevet afbrudt.\n"
"Din originale sources.list blev gemt i /etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:115
msgid "Aborting"
msgstr "Afbryder"

#: ../DistUpgrade/DistUpgradeViewText.py:120
msgid "Demoted:\n"
msgstr "Nedgraderede:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:127
msgid "To continue please press [ENTER]"
msgstr "Tryk [ENTER] for at fortsætte"

#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:193
#: ../DistUpgrade/DistUpgradeViewText.py:200
msgid "Continue [yN] "
msgstr "Fortsæt [jN] "

#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:193
msgid "Details [d]"
msgstr "Detaljer [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:159
#: ../DistUpgrade/DistUpgradeViewText.py:203
msgid "y"
msgstr "j"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:162
#: ../DistUpgrade/DistUpgradeViewText.py:210
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:170
#, python-format
msgid "No longer supported: %s\n"
msgstr "Ikke længere understøttet %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:175
#, python-format
msgid "Remove: %s\n"
msgstr "Fjern: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:185
#, python-format
msgid "Install: %s\n"
msgstr "Installér: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:190
#, python-format
msgid "Upgrade: %s\n"
msgstr "Opgradér: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:207
msgid "Continue [Yn] "
msgstr "Fortsæt [Jn] "

#: ../DistUpgrade/DistUpgradeViewText.py:228
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"For at færdiggøre opgradering er en genstart påkrævet.\n"
"Hvis du vælger \"j\" vil systemet blive genstartet."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Afbryd den igangværende opgradering?</big></b>\n"
"\n"
"Systemet kan ende i en ubrugelig tilstand, hvis du afbryder opgraderingen. "
"Det anbefales kraftigt, at du fortsætter opgraderingen."

#: ../DistUpgrade/DistUpgrade.ui.h:5
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Genstart systemet for at fuldføre opgraderingen</big></b>\n"
"\n"
"Gem dit arbejde, før du fortsætter."

#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Start opgraderingen?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "<b><big>Upgrading Ubuntu to version 11.10</big></b>"
msgstr "<b><big>Opgraderer Ubuntu til version 11.10</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "Difference between the files"
msgstr "Forskel mellem filerne"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "Distribution Upgrade"
msgstr "Distributionsopgradering"

#: ../DistUpgrade/DistUpgrade.ui.h:17
msgid "Restarting the computer"
msgstr "Genstarter computeren"

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Setting new software channels"
msgstr "Konfigurerer nye softwarekanaler"

#: ../DistUpgrade/DistUpgrade.ui.h:19
msgid "Terminal"
msgstr "Terminal"

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "_Cancel Upgrade"
msgstr "_Afbryd opgradering"

#: ../DistUpgrade/DistUpgrade.ui.h:21
msgid "_Continue"
msgstr "_Fortsæt"

#: ../DistUpgrade/DistUpgrade.ui.h:23
msgid "_Replace"
msgstr "_Erstat"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "_Report Bug"
msgstr "_Rapportér fejl"

#: ../DistUpgrade/DistUpgrade.ui.h:26
msgid "_Resume Upgrade"
msgstr "_Genoptag opgradering"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "_Start Upgrade"
msgstr "_Begynd opgradering"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:55
msgid "Please wait, this can take some time."
msgstr "Vent venligst, dette kan tage et stykke tid."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:57
msgid "Update is complete"
msgstr "Opdatering fuldført"

#: ../UpdateManager/DistUpgradeFetcher.py:108
#: ../UpdateManager/DistUpgradeFetcherKDE.py:102
msgid "Could not find the release notes"
msgstr "Kunne ikke finde udgivelsesnoterne"

#: ../UpdateManager/DistUpgradeFetcher.py:109
#: ../UpdateManager/DistUpgradeFetcherKDE.py:103
msgid "The server may be overloaded. "
msgstr "Serveren kan være overbelastet. "

#: ../UpdateManager/DistUpgradeFetcher.py:119
#: ../UpdateManager/DistUpgradeFetcherKDE.py:107
msgid "Could not download the release notes"
msgstr "Kunne ikke hente udgivelsesnoterne"

#: ../UpdateManager/DistUpgradeFetcher.py:120
#: ../UpdateManager/DistUpgradeFetcherKDE.py:108
msgid "Please check your internet connection."
msgstr "Kontrollér venligst din internetforbindelse."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:61
#: ../UpdateManager/DistUpgradeFetcherKDE.py:84
msgid "Upgrade"
msgstr "Opgradér"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:88
#: ../data/gtkbuilder/UpdateManager.ui.h:13
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Release Notes"
msgstr "Udgivelsesnoter"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:127
#: ../UpdateManager/DistUpgradeFetcherKDE.py:140
#: ../UpdateManager/DistUpgradeFetcherKDE.py:142
msgid "Downloading additional package files..."
msgstr "Henter yderligere pakkefiler..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:140
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Fil %s ud af %s ved %sB/s"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:142
#, python-format
msgid "File %s of %s"
msgstr "Fil %s ud af %s"

#: ../UpdateManager/ChangelogViewer.py:73
msgid "Open Link in Browser"
msgstr "Åbn henvisning i browser"

#: ../UpdateManager/ChangelogViewer.py:76
msgid "Copy Link to Clipboard"
msgstr "Kopiér henvisning til udklipsholder"

#: ../UpdateManager/GtkProgress.py:159
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Henter fil %(current)li af %(total)li med %(speed)s/s"

#: ../UpdateManager/GtkProgress.py:164
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Henter fil %(current)li af %(total)li"

#: ../UpdateManager/UpdateManager.py:105 ../do-release-upgrade:90
msgid "Your Ubuntu release is not supported anymore."
msgstr "Din Ubuntu-udgave understøttes ikke længere."

#: ../UpdateManager/UpdateManager.py:106
msgid ""
"You will not get any further security fixes or critical updates. Please "
"upgrade to a later version of Ubuntu Linux."
msgstr ""
"Du vil ikke modtage yderligere sikkerhedsrettelser eller kritiske "
"opdateringer. Opgrader venligst til en senere version af Ubuntu Linux."

#: ../UpdateManager/UpdateManager.py:114
msgid "Upgrade information"
msgstr "Opgraderingsinformation"

#. upload_archive = version_match.group(2).strip()
#: ../UpdateManager/UpdateManager.py:389
#, python-format
msgid "Version %s: \n"
msgstr "Udgave %s: \n"

#: ../UpdateManager/UpdateManager.py:445
msgid ""
"No network connection detected, you can not download changelog information."
msgstr ""
"Ingen netværksforbindelse fundet. Du kan ikke hente information om ændringer."

#: ../UpdateManager/UpdateManager.py:453
msgid "Downloading list of changes..."
msgstr "Henter listen med ændringer..."

#: ../UpdateManager/UpdateManager.py:497
msgid "_Deselect All"
msgstr "_Afmarkér alle"

#: ../UpdateManager/UpdateManager.py:503
msgid "Select _All"
msgstr "Markér _alt"

#: ../UpdateManager/UpdateManager.py:552
#, python-format
msgid "%(count)s update has been selected."
msgid_plural "%(count)s updates have been selected."
msgstr[0] "%(count)s opdatering er blevet valgt."
msgstr[1] "%(count)s opdateringer er blevet valgt."

#: ../UpdateManager/UpdateManager.py:556
#, python-format
msgid "%s will be downloaded."
msgstr "%s vil blive hentet."

#: ../UpdateManager/UpdateManager.py:568
msgid "The update has already been downloaded, but not installed"
msgid_plural "The updates have already been downloaded, but not installed"
msgstr[0] "Opdateringen er allerede blevet hentet, men ikke installeret"
msgstr[1] "Opdateringerne er allerede blevet hentet, men ikke installeret"

#: ../UpdateManager/UpdateManager.py:573
msgid "There are no updates to install"
msgstr "Der er ingen opdateringer at installere"

#. TRANSLATORS: this allows to switch the order of the count of
#. updates and the download size string (if needed)
#: ../UpdateManager/UpdateManager.py:579
#, python-format
msgid "%(count_str)s %(download_str)s"
msgstr "%(count_str)s %(download_str)s"

#: ../UpdateManager/UpdateManager.py:586
msgid "Unknown download size."
msgstr "Ukendt downloadstørrelse."

#: ../UpdateManager/UpdateManager.py:612
msgid ""
"It is unknown when the package information was updated last. Please click "
"the 'Check' button to update the information."
msgstr ""
"Det er uvist hvornår pakkeinformationerne sidst blev opdateret. Klik på "
"knappen \"Kontroller\" for at opdatere oplysningerne."

#: ../UpdateManager/UpdateManager.py:618
#, python-format
msgid ""
"The package information was last updated %(days_ago)s days ago.\n"
"Press the 'Check' button below to check for new software updates."
msgstr ""
"Pakkeinformationen blev sidst opdateret for %(days_ago)s dage siden.\n"
"Tryk på knappen \"Kontrollér\" nedenfor, for at søge efter nye "
"softwareopdateringer."

#: ../UpdateManager/UpdateManager.py:623
#, python-format
msgid "The package information was last updated %(days_ago)s day ago."
msgid_plural ""
"The package information was last updated %(days_ago)s days ago."
msgstr[0] ""
"Pakkeinformationen blev sidst opdateret for %(days_ago)s dag siden."
msgstr[1] ""
"Pakkeinformationen blev sidst opdateret for %(days_ago)s dage siden."

#: ../UpdateManager/UpdateManager.py:627
#, python-format
msgid "The package information was last updated %(hours_ago)s hour ago."
msgid_plural ""
"The package information was last updated %(hours_ago)s hours ago."
msgstr[0] ""
"Pakkeinformationer blev sidst opdateret for %(hours_ago)s time siden."
msgstr[1] ""
"Pakkeinformationer blev sidst opdateret for %(hours_ago)s timer siden."

#. TRANSLATORS: only in plural form, as %s minutes ago is one of 15, 30, 45 minutes ago
#: ../UpdateManager/UpdateManager.py:632 ../UpdateManager/UpdateManager.py:634
#: ../UpdateManager/UpdateManager.py:636
#, python-format
msgid "The package information was last updated about %s minutes ago."
msgstr ""
"Pakkeinformationerne blev seneste opdateret for ca. %s minutter siden."

#: ../UpdateManager/UpdateManager.py:638
msgid "The package information was just updated."
msgstr "Pakkeinformationerne er netop blevet opdateret."

#: ../UpdateManager/UpdateManager.py:658
#: ../data/gtkbuilder/UpdateManager.ui.h:22
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Softwareopdateringer retter fejl, lukker sikkerhedshuller og gør nye "
"funktioner tilgængelige."

#: ../UpdateManager/UpdateManager.py:677
msgid "Software updates may be available for your computer."
msgstr ""
"Der er muligvis tilgængelige softwareopdateringer til denne computer."

#: ../UpdateManager/UpdateManager.py:685
msgid "Welcome to Ubuntu"
msgstr "Velkommen til Ubuntu"

#: ../UpdateManager/UpdateManager.py:687
msgid ""
"These software updates have been issued since this version of Ubuntu was "
"released. If you don't want to install them now, choose \"Update Manager\" "
"from Applications later."
msgstr ""
"Disse softwareopdateringer er blevet udsendt, siden denne version af Ubuntu "
"blev udgivet. Hvis du ikke ønsker at installere dem nu, kan du på et senere "
"tidspunkt vælge \"Opdateringshåndtering\" under programmer."

#: ../UpdateManager/UpdateManager.py:689
msgid ""
"These software updates have been issued since this version of Ubuntu was "
"released. If you don't want to install them now, choose \"Update Manager\" "
"from the Administration Menu later."
msgstr ""
"Disse softwareopdateringer er blevet udsendt, siden denne version af Ubuntu "
"blev udgivet. Hvis du ikke ønsker at installere dem nu, kan du på et senere "
"tidspunkt vælge \"Opdateringshåndtering\" fra administrationsmenuen."

#: ../UpdateManager/UpdateManager.py:692
msgid "Software updates are available for this computer"
msgstr "Softwareopdateringer er tilgængelige for denne computer"

#: ../UpdateManager/UpdateManager.py:694
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from "
"Applications later."
msgstr ""
"Vælg \"Opdateringshåndtering\" fra Programmer senere, hvis du ikke ønsker at "
"installere dem nu."

#: ../UpdateManager/UpdateManager.py:696
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from the "
"Administration menu later."
msgstr ""
"Hvis ikke du ønsker at installere dem nu, kan du vælge "
"\"Opdateringshåndtering\" i menuen \"Administration\" på et senere tidspunkt."

#: ../UpdateManager/UpdateManager.py:748
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"Opgradering skal bruge %s fri plads på disk \"%s\". Frigør mindst %s disk "
"plads på \"%s\". Tøm papirkurven og fjern midlertidige pakker fra tidligere "
"installationer ved at bruge \"sudo apt-get clean\"."

#: ../UpdateManager/UpdateManager.py:773
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr ""
"Computeren kræver en genstart for at færdiggøre installationen af "
"opdateringer. Gem venligst dit arbejde før du fortsætter."

#: ../UpdateManager/UpdateManager.py:837
msgid "Reading package information"
msgstr "Læser pakkeinformation"

#: ../UpdateManager/UpdateManager.py:852
msgid "Connecting..."
msgstr "Forbinder..."

#: ../UpdateManager/UpdateManager.py:869
msgid "You may not be able to check for updates or download new updates."
msgstr ""
"Du vil muligvis ikke kunne søge efter opdateringer eller hente nye "
"opdateringer."

#: ../UpdateManager/UpdateManager.py:994
msgid "Could not initialize the package information"
msgstr "Kunne ikke initialisere pakkeinformationen"

#: ../UpdateManager/UpdateManager.py:995
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"Under initialisering af pakkeinformationen, opstod der et problem, som ikke "
"kunne løses.\n"
"\n"
"Rapportér venligst denne fejl til \"update-manager\"-pakken og inkludér "
"følgende fejlmeddelelse:\n"

#: ../UpdateManager/UpdateManager.py:1024
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"Der opstod et problem, som ikke kunne løses, under beregning af "
"opgraderingen.\n"
"\n"
"Rapportér venligst denne fejl til \"update-manager\"-pakken og inkludér "
"følgende fejlmeddelelse:"

#: ../UpdateManager/UpdateManager.py:1049
msgid " (New install)"
msgstr " (Ny installation)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:1056
#, python-format
msgid "(Size: %s)"
msgstr "(Størrelse: %s)"

#: ../UpdateManager/UpdateManager.py:1058
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Fra udgave %(old_version)s til %(new_version)s"

#: ../UpdateManager/UpdateManager.py:1062
#, python-format
msgid "Version %s"
msgstr "Udgave %s"

#: ../UpdateManager/UpdateManager.py:1093 ../do-release-upgrade:102
msgid "Release upgrade not possible right now"
msgstr "Udgivelsesopgradering er ikke mulig i øjeblikket"

#: ../UpdateManager/UpdateManager.py:1094 ../do-release-upgrade:103
#, c-format, python-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Opgraderingen af udgivelsen kan ikke udføres i øjeblikket. Prøv venligst "
"igen senere. Serveren svarede: \"%s\""

#: ../UpdateManager/UpdateManager.py:1096 ../check-new-release-gtk:116
msgid "Downloading the release upgrade tool"
msgstr "Hent værktøjet til udgivelsesopgraderingen"

#: ../UpdateManager/UpdateManager.py:1103
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Ny Ubuntu-version \"%s\" er tilgængelig</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1142
msgid "Software index is broken"
msgstr "Softwareindeks er i stykker"

#: ../UpdateManager/UpdateManager.py:1143
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Det er umuligt at installere eller fjerne software. Brug venligst "
"pakkehåndteringsprogrammet \"Synaptic\" eller kør \"sudo apt-get install -"
"f\" i en terminal for at ordne dette problem først."

#: ../UpdateManager/UnitySupport.py:57
msgid "Check for Updates"
msgstr "Søg efter opdateringer"

#: ../UpdateManager/UnitySupport.py:66
msgid "Install All Available Updates"
msgstr "Installér alle tilgængelige opdateringer"

#: ../UpdateManagerText/UpdateManagerText.py:31
msgid "Cancel"
msgstr "Annullér"

#: ../UpdateManagerText/UpdateManagerText.py:32
msgid "Install"
msgstr "Installér"

#: ../UpdateManagerText/UpdateManagerText.py:50
msgid "Building Updates List"
msgstr "Opbygger opdateringsliste"

#: ../UpdateManagerText/UpdateManagerText.py:53
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"En normal opgradering kan ikke beregnes. Kør venligst: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
" * Dette kan være af følgende grunde:\n"
" * En tidligere opgradering, der ikke blev fuldført\n"
" * Problemer med noget af det installerede software\n"
" * Uofficielle softwarepakker, der ikke leveres af Ubuntu\n"
" * Normale ændringer i en udviklingsudgave af Ubuntu"

#: ../UpdateManagerText/UpdateManagerText.py:122
msgid "Downloading changelog"
msgstr "Henter listen med ændringer"

#: ../UpdateManager/Core/MyCache.py:128
#, python-format
msgid "Other updates (%s)"
msgstr "Andre opdateringer (%s)"

#: ../UpdateManager/Core/MyCache.py:283
#, python-format
msgid ""
"Changes for the versions:\n"
"Installed version: %s\n"
"Available version: %s\n"
"\n"
msgstr ""
"Ændringer for versionerne:\n"
"Installerede version: %s\n"
"Tilgængelige version: %s\n"
"\n"

#: ../UpdateManager/Core/MyCache.py:297
msgid "This update does not come from a source that supports changelogs."
msgstr ""
"Denne opdatering kommer ikke fra en kilde, der understøtter ændringslogge."

#: ../UpdateManager/Core/MyCache.py:303 ../UpdateManager/Core/MyCache.py:324
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Fejl ved hentning af ændringslisten.\n"
"Kontrollér venligst din internetforbindelse."

#: ../UpdateManager/Core/MyCache.py:313
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Listen med ændringer indeholder ikke nogen relevante ændringer.\n"
"\n"
"Benyt venligst http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"indtil ændringerne bliver tilgængelige eller prøv igen senere."

#: ../UpdateManager/Core/MyCache.py:318
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Listen med ændringer er ikke tilgængelig endnu.\n"
"\n"
"Brug http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"indtil ændringerne bliver tilgængelige eller prøv igen senere."

#: ../UpdateManager/Core/UpdateList.py:49
msgid "Failed to detect distribution"
msgstr "Kunne ikke fastslå distribution"

#: ../UpdateManager/Core/UpdateList.py:50
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr ""
"Der opstod en fejl \"%s\", mens det blev undersøgt, hvilket system du bruger."

#: ../UpdateManager/Core/UpdateList.py:61
msgid "Important security updates"
msgstr "Vigtige sikkerhedsopdateringer"

#: ../UpdateManager/Core/UpdateList.py:62
msgid "Recommended updates"
msgstr "Anbefalede opdateringer"

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Proposed updates"
msgstr "Foreslåede opdateringer"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Backports"
msgstr "Tilbageporteringer"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Distribution updates"
msgstr "Distributionsopdateringer"

#: ../UpdateManager/Core/UpdateList.py:70
msgid "Other updates"
msgstr "Andre opdateringer"

#: ../data/gtkbuilder/UpdateManager.ui.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>Du skal manuelt søge efter opdateringer</big></b>\n"
"\n"
"Dit system søger ikke automatisk efter nye opdateringer. Du kan indstille "
"denne adfærd i <i>Softwarekilder</i> under <i>Opdateringer</i>-fanebladet."

#: ../data/gtkbuilder/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Ikke alle opdateringer kan installeres</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:5
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Kører på batteri</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:6
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<big><b>Starter opdateringshåndtering</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:7
msgid "Changes"
msgstr "Ændringer"

#: ../data/gtkbuilder/UpdateManager.ui.h:8
msgid "Chec_k"
msgstr "_Kontrollér"

#: ../data/gtkbuilder/UpdateManager.ui.h:9
msgid "Co_ntinue"
msgstr "F_ortsæt"

#: ../data/gtkbuilder/UpdateManager.ui.h:10
msgid "Description"
msgstr "Beskrivelse"

#: ../data/gtkbuilder/UpdateManager.ui.h:11
msgid "Description of update"
msgstr "Beskrivelse af opdatering"

#: ../data/gtkbuilder/UpdateManager.ui.h:12
msgid "It’s safer to connect the computer to AC power before updating."
msgstr ""
"Det er sikrere at tilslutte computeren ekstern strømforsyning før "
"opdateringen."

#: ../data/gtkbuilder/UpdateManager.ui.h:14
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Kør en delvis opgradering for at installere så mange opdateringer, som "
"muligt. \n"
"\n"
"Dette kan skyldes:\n"
"* En tidligere opgradering, som ikke blev fuldført\n"
"* Problemer med dele af den installerede software\n"
"* Uofficielle softwarepakker, som ikke er leveret af Ubuntu\n"
"* Normale ændringer i en før udgivelse-version af Ubuntu"

#: ../data/gtkbuilder/UpdateManager.ui.h:21
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "Show progress of individual files"
msgstr "Vis fremgang for individuelle filer"

#: ../data/gtkbuilder/UpdateManager.ui.h:23
msgid "Starting Update Manager"
msgstr "Starter opdateringshåndtering"

#: ../data/gtkbuilder/UpdateManager.ui.h:24
msgid "U_pgrade"
msgstr "_Opgradér"

#: ../data/gtkbuilder/UpdateManager.ui.h:25
#: ../data/update-manager.desktop.in.h:3
msgid "Update Manager"
msgstr "Opdateringshåndtering"

#: ../data/gtkbuilder/UpdateManager.ui.h:26
msgid "You are connected via a wireless modem."
msgstr "Du er forbundet via et trådløst modem."

#: ../data/gtkbuilder/UpdateManager.ui.h:27
msgid ""
"You are connected via roaming and may be charged for the data consumed by "
"this update."
msgstr ""
"Du er forbundet via roaming og kan blive takseret for dataforbruget ved "
"denne opdatering."

#: ../data/gtkbuilder/UpdateManager.ui.h:28
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr "Dit system kører på batteri. Er du sikker på, at du vil fortsætte?"

#: ../data/gtkbuilder/UpdateManager.ui.h:29
msgid "_Hide this information in the future"
msgstr "_Skjul denne information for fremtiden"

#: ../data/gtkbuilder/UpdateManager.ui.h:30
msgid "_Install Updates"
msgstr "_Installér opdateringer"

#: ../data/gtkbuilder/UpdateManager.ui.h:31
msgid "_Partial Upgrade"
msgstr "_Delvis opgradering"

#: ../data/gtkbuilder/UpdateManager.ui.h:33
msgid "_Settings..."
msgstr "_Opsætning..."

#: ../data/gtkbuilder/UpdateManager.ui.h:34
msgid "_Upgrade"
msgstr "_Opgradér"

#: ../data/gtkbuilder/UpdateManager.ui.h:35
msgid "updates"
msgstr "opdateringer"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>En ny udgave af Ubuntu er tilgængelig. Ønsker du at opgradere?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Ask Me Later"
msgstr "Spørg mig senere"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "Opgradér ikke"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid "Yes, Upgrade Now"
msgstr "Ja, opgradér nu"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid ""
"You can upgrade at a later time by opening Update Manager and click on "
"\"Upgrade\"."
msgstr ""
"Du kan opgradere på ethvert senere tidspunkt ved at åbne "
"opdateringshåndteringen og klikke på \"Opgradér\"."

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:8
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Du har afslået at opgradere til den nye Ubuntu"

#: ../data/update-manager.desktop.in.h:1
msgid "Show and install available updates"
msgstr "Vis og installér tilgængelige opdateringer"

#: ../data/update-manager.desktop.in.h:2
msgid "Software Updates"
msgstr "Softwareopdateringer"

#: ../update-manager:54 ../do-release-upgrade:41
msgid "Show version and exit"
msgstr "Vis version og afslut"

#: ../update-manager:57
msgid "Directory that contains the data files"
msgstr "Mappe, der indeholder datafilerne"

#: ../update-manager:60
msgid "Check if a new Ubuntu release is available"
msgstr "Kontroller, om der er en ny Ubuntu-udgave tilgængelig"

#: ../update-manager:63 ../do-release-upgrade:44 ../check-new-release-gtk:176
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Undersøg om det er muligt at opgradere til den seneste udviklerudgivelse"

#: ../update-manager:67
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr ""
"Opgradér ved brug af den seneste foreslåede udgave af udgivelsesopgradering"

#: ../update-manager:74
msgid "Do not focus on map when starting"
msgstr "Fokusér ikke på kortet under opstarten"

#: ../update-manager:77
msgid "Try to run a dist-upgrade"
msgstr "Forsøg at udføre en distributionsopgradering (dist-upgrade)"

#: ../update-manager:80
msgid "Do not check for updates when starting"
msgstr "Søg ikke efter opdateringer ved opstart"

#: ../update-manager:84 ../do-release-upgrade:60
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Udfør en testopgradering med et aufs-testtilstandslag"

#: ../update-manager:104
msgid "Running partial upgrade"
msgstr "Udfører delvis opgradring"

#: ../do-release-upgrade:48 ../check-new-release-gtk:180
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Prøv at opgradere til den seneste udgivelse ved hjælp af "
"opgraderingsprogrammet fra $distro-proposed"

#: ../do-release-upgrade:52
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Kør i en speciel opgraderingstilstand.\n"
"I øjeblikket understøttes \"desktop\" til almindelig opgradering af en "
"hjemmecomputer og \"server\" til servere."

#: ../do-release-upgrade:58
msgid "Run the specified frontend"
msgstr "Kør den specificerede brugergrænseflade"

#: ../do-release-upgrade:63
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Kontrollér kun om en ny distributionsudgave er tilgængelig, og rapportér "
"resultatet via afslutningskoden"

#: ../do-release-upgrade:77
msgid "Checking for a new ubuntu release"
msgstr "Undersøg om der er nye Ubuntu-udgivelser"

#: ../do-release-upgrade:91
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Se venligst følgende link for opgraderingsinformation:\n"
"%(url)s\n"

#: ../do-release-upgrade:97
msgid "No new release found"
msgstr "Ingen ny udgave fundet"

#: ../do-release-upgrade:109
#, c-format
msgid "New release '%s' available."
msgstr "Ny udgivelse \"%s\" tilgængelig."

#: ../do-release-upgrade:110
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Kør \"do-release-upgrade\" for at opgradere til den."

#: ../check-new-release-gtk:91
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Opgradering til Ubuntu %(version)s er tilgængelig"

#: ../check-new-release-gtk:132
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Du har afslået at opgradere til Ubuntu %s"

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr "Ikke-implementeret handling: %s"

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr "En fil på disken"

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr ".deb-pakke"

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr "Installér manglende pakke."

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr "Pakke %s bør installeres."

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "%i forældede emner i statusfilen"

#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr "Forældede emner i dpkg-status"

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr "Forældede dpkg-statuslinjer"

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"Hvis kdelibs4-dev er installeret når der opgraderes, skal kdelibs5-dev "
"installeres. Se bugs.launchpad.net, fejl #279.621 for yderligere oplysninger."

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "%s skal være markeret som manuelt installeret."

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"Fjern lilo, da grub også er installeret. (Se fejl #314004 for yderligere "
"oplysninger.)"

#~ msgid "0 KB"
#~ msgstr "0 KB"

#~ msgid "The window size"
#~ msgstr "Vinduesstørrelsen"

#~ msgid "Show details of an update"
#~ msgstr "Vis detaljer for opdatering"

#~ msgid "The package information was last updated less than one hour ago."
#~ msgstr ""
#~ "Pakkeinformationerne blev sidst opdateret for mindre end en time siden."

#~ msgid "_Check All"
#~ msgstr "_Afkryds alle"

#~ msgid "_Uncheck All"
#~ msgstr "_Fjern alle afkrydsninger"

#~ msgid ""
#~ "It was impossible to install a required package. Please report this as a "
#~ "bug. "
#~ msgstr ""
#~ "Det var umuligt at installere en påkrævet pakke. Rapporter venligst dette "
#~ "som en fejl. "

#, python-format
#~ msgid "%.0f KB"
#~ msgstr "%.0f KB"

#~ msgid "1 KB"
#~ msgstr "1 KB"

#~ msgid "*DEPRECATED* this option will be ignore"
#~ msgstr "*FORÆLDET* denne mulighed vil blive ignoreret"

#~ msgid ""
#~ "If none of this applies, then please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Hvis intet af dette passer, bedes du rapportere denne som en fejl i \"update-"
#~ "manager\"-pakken og inkludere filerne i /var/log/dist-upgrade/ i "
#~ "fejlrapporten."

#~ msgid ""
#~ "Upgrading the repository information resulted in a invalid file. Please "
#~ "report this as a bug."
#~ msgstr ""
#~ "Opgradering af arkivet resulterede i en ødelagt fil. Rapportér venligst "
#~ "dette som en fejl."

#, python-format
#~ msgid ""
#~ "\n"
#~ "\n"
#~ "Please report this bug against the 'update-manager' package and include the "
#~ "files in /var/log/dist-upgrade/ in the bug report.\n"
#~ "%s"
#~ msgstr ""
#~ "\n"
#~ "\n"
#~ "Rapportér venligst denne fejl for \"update-manager\"-pakken og inkludér "
#~ "filerne i /var/log/dist-upgrade/ i fejlrapporten.\n"
#~ "%s"

#~ msgid ""
#~ "Preparing the system for the upgrade failed. Please report this as a bug "
#~ "against the 'update-manager' package and include the files in /var/log/dist-"
#~ "upgrade/ in the bug report."
#~ msgstr ""
#~ "Klargøring af systemet til opgraderingen mislykkedes. Du bedes rapportere "
#~ "dette som en fejl for \"update-manager\"-pakken og inkludere filerne i "
#~ "/var/log/dist-upgrade/ i fejlrapporten."

#~ msgid ""
#~ "The system was unable to get the prerequisites for the upgrade. The upgrade "
#~ "will abort now and restore the original system state.\n"
#~ "\n"
#~ "Please report this as a bug against the 'update-manager' package and include "
#~ "the files in /var/log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "Systemet var ude af stand til at hente forudsætningerne for opgraderingen. "
#~ "Opgraderingen vil blive afbrudt nu og gendanne den oprindelige "
#~ "systemtilstand.\n"
#~ "\n"
#~ "Du bedes rapportere dette som en fejl for \"update-manager\"-pakken og "
#~ "inkludere filerne i /var/log/dist-upgrade/ i fejlrapporten."

#~ msgid ""
#~ "This is most likely a bug in the upgrade tool. Please report it as a bug"
#~ msgstr ""
#~ "Dette er højst sandsynligt en fejl i opgraderingsværktøjet. Rapportér "
#~ "venligst dette som en fejl"

#~ msgid ""
#~ "This usually is caused by a system were /tmp is mounted noexec. Please "
#~ "remount without noexec and run the upgrade again."
#~ msgstr ""
#~ "Dette skyldes oftest et system, hvor /tmp er monteret som noexec. Genmontér "
#~ "uden noexec og kør opgraderingen igen."

#~ msgid ""
#~ "It is unknown when the package information was updated last. Please try "
#~ "clicking on the 'Check' button to update the information."
#~ msgstr ""
#~ "Det er uvist, hvornår pakkeinformationerne sidst blev opdateret. Klik "
#~ "venligst \"Kontrollér\"-knappen for at opdatere informationen."

# Man udsender opdateringer, men man udsteder pas (tilladelser)
#~ msgid ""
#~ "These software updates have been issued since Ubuntu was released. If you "
#~ "don't want to install them now, choose \"Update Manager\" from the "
#~ "Administration Menu later."
#~ msgstr ""
#~ "Disse opdateringer er blevet udsendt, siden Ubuntu blev frigivet. Hvis du "
#~ "ikke ønsker at installere dem nu, kan du vælge \"Opdateringshåndtering\" i "
#~ "menuen \"Administration\" på et senere tidspunkt."

#~ msgid "Previous selected"
#~ msgstr "Tidligere valgte"

#~ msgid "*deprecated* Check for new distribution releases"
#~ msgstr "*forældet* Undersøg om der er nye distributionsudgivelser"

#~ msgid "Auto close the install window after successful install"
#~ msgstr "Luk automatisk installationsvinduet efter vellykket installation"

#~ msgid "First run welcome message"
#~ msgstr "Velkomstmeddelelse ved først opstart"

#~ msgid ""
#~ "If automatic checking for updates is disabled, you have to reload the "
#~ "channel list manually. This option allows to hide the reminder shown in this "
#~ "case."
#~ msgstr ""
#~ "Hvis søg automatisk efter opdateringer er slået fra, er du nødt til at "
#~ "genindlæse kanallisten manuelt. Denne indstilling tillader, at påmindelsen "
#~ "skjules i dette tilfælde."

#~ msgid "If this key is set a first run welcome message will be presented."
#~ msgstr ""
#~ "Hvis denne indstilling er valgt, vil en velkomstmeddelelse blive vist ved "
#~ "den første opstart."

#~ msgid ""
#~ "If this key is set the install window will be automatically closed on "
#~ "successful installation."
#~ msgstr ""
#~ "Hvis denne indstilling er valgt, vil installationsvinduet automatisk lukke "
#~ "efter vellykket installation."

#~ msgid ""
#~ "If this key is set the main update list window will show version information "
#~ "(from version to version)."
#~ msgstr ""
#~ "Hvis denne indstilling er valgt, vil det hovedvinduet med opdateringslisten "
#~ "vise versionsinformation (fra version til version)."

#~ msgid "Remind to reload the channel list"
#~ msgstr "Påmind om at genindlæse kanallisten"

#~ msgid "Show version in update list"
#~ msgstr "Vis version i opdateringsliste"

#~ msgid ""
#~ "Stores the state of the expander that contains the list of changes and the "
#~ "description"
#~ msgstr ""
#~ "Gemmer tilstand for udfolderen, der indeholder listen med ændringer og "
#~ "beskrivelsen"

#~ msgid ""
#~ "This key is deprecated in favor of the file /etc/update-manager/release-"
#~ "upgrades Check automatically if a new version of the current distribution is "
#~ "available and offer to upgrade (if possible)."
#~ msgstr ""
#~ "Denne nøgle er forældet til fordel for filen /etc/update-manager/release-"
#~ "upgrades, som automatisk undersøger, om der er en ny udgave af den nuværende "
#~ "distribution tilgængelig og tilbyder at opgradere (hvis det er muligt)."

#, python-format
#~ msgid ""
#~ "Changes for the versions:\n"
#~ "%s\n"
#~ "%s\n"
#~ "\n"
#~ msgstr ""
#~ "Ændringer for versionerne:\n"
#~ "%s\n"
#~ "%s\n"
#~ "\n"

#, python-format
#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore.\n"
#~ "This indicates a serious error, please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Den essentielle pakke \"%s\" kan ikke længere findes, efter din "
#~ "pakkeinformation er blevet opdateret.\n"
#~ "Dette tyder på en alvorlig fejl. Du bedes rapportere denne fejl mod \"update-"
#~ "manager\"-pakken og inkludere filerne i /var/log/dist-upgrade/ i "
#~ "fejlrapporten."

#~ msgid "This change is not coming from a source that supports changelogs."
#~ msgstr ""
#~ "Denne ændring kommer ikke fra en kilde, der understøtter ændringslogge."

#~ msgid "Settings..."
#~ msgstr "Indstillinger..."

#~ msgid "Stores the size of the update-manager dialog"
#~ msgstr "Gemmer størrelsen på dialog eller opdateringshåndteringensdialogen"

#~ msgid ""
#~ "This will permanently hide the new release prompt from check-new-release-"
#~ "gtk. Note that the small button in the main update-manager UI will still be "
#~ "there."
#~ msgstr ""
#~ "Dette vil permanent skjule den nye udgivelsesprompt fra check-new-release-"
#~ "gtk. Bemærk at den lille knap i grænsefladen for opdateringshåndteringen "
#~ "stadig vil være til stede."

#~ msgid "make check-new-release-gtk ignore a given new release"
#~ msgstr "få check-new-release-gtk til at ignorere en given ny udgave"

#~ msgid ""
#~ "The upgrade has aborted. Please check your Internet connection or "
#~ "installation media and try again. All files downloaded so far are kept."
#~ msgstr ""
#~ "Opdateringen er blevet afbrudt. Kontroller venligst din internetforbindelse "
#~ "og installationsmedie og prøv igen. Alle filer der blev hentet beholdes."

#~ msgid "Show package summary before package name in update list"
#~ msgstr "Vis pakkebeskrivelse før pakkenavn i opdateringslisten"

#~ msgid ""
#~ "If this key is set, the main update list window will show the package "
#~ "summary before the package name."
#~ msgstr ""
#~ "Hvis denne værdi er sat, vil hovedvinduet for opdateringslisten vise "
#~ "pakkebeskrivelsen før pakkenavnet."

#~ msgid "<b><big>Upgrading Ubuntu to version 11.04</big></b>"
#~ msgstr "<b><big>Opgraderer Ubuntu til version 11.04</big></b>"

#~ msgid ""
#~ "These software updates have been issued since Ubuntu was released. If you "
#~ "don't want to install them now, choose \"Update Manager\" from Applications "
#~ "later."
#~ msgstr ""
#~ "Disse programopdateringer er blevet udsendt siden Ubuntu blev udgivet. Vælg "
#~ "\"Opdateringshåndtering\" fra Programmer senere, hvis du ikke ønsker at "
#~ "installere dem nu."
